assembly/std/index.ts

```diff
@@ -331,18 +458,28 @@ export function isSignatureValid(
 }
 
 /**
- * Converts a public key to an address
+ * Retrieves an 'Address' object from the given public key.
+ *
+ * @param pubKey - Base58check encoded public key of the address
+ *
+ * @returns the fetched address as an 'Address' object.
+ *
+ * @throws
+ * - if the public key is invalid
  *
- * @param pubKey - Base58check encoded
  */
 export function publicKeyToAddress(pubKey: string): Address {
```

Comment:
Can this function go in the Address class as a static function ?

Reply:
For now we can leave it like that.